HACKING.md

```diff
@@ -47,34 +50,13 @@ It is a good idea to
 and 
 [set the kubernetes plugin to use DNS for service names](TROUBLESHOOTING.md#issue-for-jenkins-dns-names).
```

Comment:
are the statements above needed any longer?

Reply:
I deleted the one about updating the plugins but kept the DNS one. I haven't hit it myself, though I don't think switching to s2i would somehow fix that. Eventually, we could likely tweak the s2i to include this as part of the built master image.